apps/build.sh

```diff
@@ -223,6 +223,11 @@ for x in $IMAGES ; do
 		echo "osftoken not provided, skipping publishing step"
 	fi
 
+	status "Doing a syft SBOM scan"
+	sbom_dst=/archive/sboms/${ct_base}/${ARCH}.sdpx.json
+	mkdir -p $(dirname $sbom_dst)
+	syft ${ct_base}:$TAG-$ARCH -o spdx-json > $sbom_dst
```

Comment:
How long does it take to scan an image? If it takes some time I wonder if it's possible to output some progress.

Reply:
it sends progress to stderr